### PR TITLE
Game logic

### PR DESCRIPTION
**Pull request by Alaska01**
Create the game logic for a tic-tac-toe game, with the game logic in the lib folder and the puts comments where in the main.rb file in the bin folder. This pull request was marked as invalid : https://github.com/Daniduran-dev/tic-tac-toe/pull/6, hence I had to create another pull request and resubmit again.

### CODE REVIEW

File: bin/main.rb [hunk 1]
```diff
@@ -1,250 +1,74 @@
...
-
-# Run the game.
-# The above is the pseudo code to build a working tic tac toe user interface.
-
-# REAL USER INTERFACE FOR TIC TAC TOE TESTED BELOW
-
-class TicTacToe
-  attr_accessor :board, :player
-  # initialize some variable that can be accessed throughout the game
+require_relative '..\lib\board'
```
Comment (addod19) [inline bin/main.rb]: Kindly replace `..\lib\board` with `../lib/board`, what we have currently gives a `loadError`. Let us fix it

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @Alaska01, great job with this project :+1: The app works as expected, the code is clean :smile: Great job, before we get our project approved, let us address the comments below:

- Kindly enable linter checks for the repo. Currently, linter check is not working
- Update the executable file to use forward slash `/` instead of back slash `\`

KIndly make these changes and submit another code review :+1: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Review by maelfosso — APPROVED** [review]
# Congratulations :clap: 

Thank you for applying the suggested changes. You can merge it now :+1: 

Good continuation :handshake: 

Mael FOSSO

**Comment by Alaska01** [comment]
Project approved, merging to go to the next milestone!!!!

**Pull request merged**